pegasus/sites.v3/code.org/views/dance_tutorial_section.haml

```diff
@@ -1,5 +1,9 @@
-.dance-subheading
-  %a{href: "https://code.org/translate/dance"}= I18n.t(:hoc2018_dance_party_subheading)
+%div.dance-text-icon-container
+  %a{href: "https://code.org/translate/dance"}
+    %div.dance-translate-link-text
+      =I18n.t(:hoc2018_dance_party_subheading)
+  %a.dance-chevron-icon{href: "https://code.org/translate/dance"}
```

Comment:
There don't need to be two a tags.  Put the text and the icon in the same  parent a tag.